Packs/CommunityCommonScripts/Scripts/DateTimeNowToEpoch/DateTimeNowToEpoch.py

```diff
@@ -0,0 +1,15 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def retrieve_epoch() -> int:
+    # Return current epoch value for datetime.now
+    return int(time.time())
+
+
+def main():
+    demisto.results(retrieve_epoch)
```

Comment:
Have you tired to use return_results() instead? in order to use the retrieve_epoch value?

Reply:
Done